windows/README_MINGW

```diff
@@ -0,0 +1,268 @@
+
+Make sure gcc runs and it is the expected version:
+
+$ gcc --version
+gcc.exe (x86_64-posix-sjlj-rev0, Built by MinGW-W64 project) {version}
+
+
+
+Preparing JACK dependencies
+---------------------------
+
+There are prebuilt MinGW binaries for all the libraries that can be installed
+using the pacman package manager, but since we are using a compiler that is not
+the default version shipped by MinGW, it seems better idea to build from source
+to avoid any linker and runtime issues. A good technical explanation and/or
+evidence for this statement is currently missing from this guide.
+
+Fortunately there are PKGBUILD files for doing so together with a nice guide at
+https://github.com/msys2/MINGW-packages
+
+git clone https://github.com/msys2/MINGW-packages.git
+
+Before building libraries, adjust the includes path:
+
+export C_INCLUDE_PATH=/mingw64/include
+
+The basic procedure for building and installing libraries is:
+
+cd MINGW-packages/mingw-w64-{libname}
+MINGW_INSTALLS=mingw64 makepkg-mingw -sLf
+pacman -U mingw-w64-{libname}-{suffix}.pkg.tar.xz
+
+Repeat the procedure for each library listed below replacing {libname} with the
+appropriate name and {suffix} with whatever the above process creates. Keep the
+recommended build order, for example libsamplerate depends on libsndfile, and
+libsystre depends on libtre-git. libsystre is a wrapper around libtre that
+allows including <regex.h> later.
+
+Some libraries like libsndfile and libsamplerate will ask for installing extra
+dependencies, it is ok to do so.
+
+For low latency audio it is recommended to build portaudio with ASIO support, so
+the Steinberg ASIO SDK should be manually downloaded beforehand. It can be found
+at https://www.steinberg.net/en/company/developers.html. The waf script will
+later check if the SDK is present at /opt/asiosdk
+
+wget https://www.steinberg.net/asiosdk -O /tmp/asiosdk.zip
+unzip /tmp/asiosdk.zip -d /tmp
+mkdir /opt   <- MinGW does not create /opt during installation
+mv /tmp/asiosdk_{version}_{date} /opt/asiosdk
+
+The description file in portaudio (MINGW-packages/mingw-w64-portaudio/PKGBUILD)
+needs to be patched so configure is called with the necessary flags for ASIO:
+
+--with-asiodir=/opt/asiosdk   <- new option     
+--with-winapi=wmme,directx,wasapi,vdmks,asio   <- append 'asio' to existing
+
+Both static and shared library versions of portaudio are built. To prevent
+errors while building the shared version, also insert the following lines in
+PKGBUILD after the first make call (between lines 60 and 61):
+
+find /opt/asiosdk -name "*.lo" -type f -delete
+find /opt/asiosdk -name "*.o" -type f -delete
+
+Finally here is the list of libraries to build:
+
+db
+libsndfile
+libsamplerate
+libtre-git
+libsystre
+portaudio
+
+
+
+Compiling JACK
+--------------
+
+- Clone repo
+
+git clone https://github.com/jackaudio/jack2
+
+- Build and install
+
+cd jack2
+./waf configure --prefix=/opt/jack && ./waf -p install
```

Comment:
Note to possibly add for a mingw build.  
The `./waf configure .....`  was not finding the new built libraries.  I had an msys64 pkg-config , `which pkg-config`   showed `/usr/bin/pkg-config`   if i ran ` /mingw64/bin/pkg-config --list-all` it showed all the built libraries. 
My fix  was to ensure mingw pkg-config was installed `pacman -S mingw-w64-x86_64--pkg-config`   and to remove msys pkg-config `pacman -R pkg-config`

Reply:
Can you submit a patch for this and the previous (possible) issue?